primitives/src/lib.rs

```diff
@@ -16,8 +19,8 @@ pub mod v0 {
 	mod error {
 		use super::*;
 
-		/// Reason why a Pop API call failed.
-		#[derive(Encode, Decode, Debug, Eq, PartialEq)]
+		/// Reason why a call failed.
+		#[derive(Encode, Decode, Debug, Eq, PartialEq, Clone)]
```

Comment:
Should we feature gate `Clone` for only `cfg(test)` as well?

Reply:
Outdated